pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -177,12 +177,40 @@ class TestToDatetime(object):
 
     def test_to_datetime_iso_week_year_format(self):
         data = [
-            ['2015-53-1', '%G-%V-%u',
-            datetime(2015, 12, 28, 0, 0)]
+            ['2015-1-1', '%G-%V-%u',
+            datetime(2014, 12, 29, 0, 0)], #negative ordinal (date in previous year)
+            ['2015-1-4', '%G-%V-%u',
+            datetime(2015, 1, 1, 0, 0)],
+            ['2015-1-7', '%G-%V-%u',
+            datetime(2015, 1, 4, 0, 0)] 
             ]
         for s, format, dt in data:
             assert to_datetime(s, format=format) == dt
 
+    def test_ValueError_iso_week_year(self):
+        # 1. ISO week (%V) is specified, but the year is specified with %Y
+        # instead of %G
+        with pytest.raises(ValueError):
```

Comment:
Let's use `tm.assert_raises_regex` instead.  It is stronger to test not only the Exception raised but also the message that comes along with it.